lib/auth/auth.go

```diff
@@ -298,6 +320,18 @@ func (a *Server) runPeriodicOperations() {
 					log.Errorf("Failed to perform cert rotation check: %v.", err)
 				}
 			}
+		case <-heartbeatCheckTicker.Next():
+			nodes, err := a.GetNodes(defaults.Namespace, services.SkipValidation())
+			if err != nil {
+				log.Errorf("Failed to update prometheus metric: %v", err)
```

Comment:
Be more specific here (e.g. `Failed to load nodes for heartbeat metric calculation`).